<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>painter</class>
 <widget class="QMainWindow" name="painter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>552</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Style</string>
    </property>
    <addaction name="actionPen"/>
    <addaction name="actionBrush"/>
    <addaction name="actionColor"/>
    <addaction name="actionRectangle"/>
    <addaction name="actionEllipse"/>
   </widget>
   <widget class="QMenu" name="menuArchino">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionSave"/>
    <addaction name="actionOpen"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="SizeEdit"/>
    <addaction name="actionErase"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout_Painter"/>
   </widget>
   <addaction name="menuArchino"/>
   <addaction name="menuMenu"/>
   <addaction name="menuEdit"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionPen"/>
   <addaction name="actionBrush"/>
   <addaction name="actionColor"/>
   <addaction name="actionEllipse"/>
   <addaction name="actionRectangle"/>
   <addaction name="SizeEdit"/>
   <addaction name="actionErase"/>
   <addaction name="actionAbout_Painter"/>
  </widget>
  <action name="SizeEdit">
   <property name="icon">
    <iconset>
     <normalon>:/new/Images/Images/pensize.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>SIze</string>
   </property>
  </action>
  <action name="actionColor">
   <property name="icon">
    <iconset>
     <normalon>:/new/Images/Images/color_swatch.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Color</string>
   </property>
  </action>
  <action name="actionErase">
   <property name="icon">
    <iconset>
     <normalon>:/new/Images/Images/erase.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Erase</string>
   </property>
  </action>
  <action name="actionRectangle">
   <property name="icon">
    <iconset>
     <normalon>:/new/Images/Images/shape_rectangle.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Rectangle</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="PainterImages.qrc">
     <normaloff>:/new/Images/Images/save.png</normaloff>
     <normalon>Images/page_save.png</normalon>:/new/Images/Images/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset>
     <normalon>:/new/Images/Images/door_open.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionEllipse">
   <property name="icon">
    <iconset>
     <normalon>:/new/Images/Images/ellipse.jpg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Ellipse</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="PainterImages.qrc">
     <normaloff>:/new/Images/Images/add.png</normaloff>
     <normalon>Images/add.png</normalon>:/new/Images/Images/add.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionAbout_Painter">
   <property name="icon">
    <iconset>
     <normalon>:/new/Images/Images/About_Icon_1.jpg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>About Painter</string>
   </property>
  </action>
  <action name="actionPen">
   <property name="icon">
    <iconset>
     <normalon>:/new/Images/Images/Pen.jpg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Pen</string>
   </property>
  </action>
  <action name="actionBrush">
   <property name="icon">
    <iconset>
     <normalon>:/new/Images/Images/Brush.jpg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Brush</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="PainterImages.qrc"/>
 </resources>
 <connections/>
</ui>
